pkg/segment/aggregations/segaggs.go

```diff
@@ -168,7 +168,36 @@ RenamingLoop:
 	}
 
 	if colReq.RenameColumns != nil {
-		return errors.New("performColumnsRequest: processing ColumnsRequest.RenameColumns is not implemented")
+		// return errors.New("performColumnsRequest: processing ColumnsRequest.RenameColumns is not implemented")
+
```

Comment:
nit: delete this instead of commenting it out

Reply:
Sure, got it!